CONTRIBUTING.md

```diff
@@ -45,23 +50,31 @@ Here are the current repos for the gonum project.  If code you want to contribut
 * [talks](https://github.com/gonum/talks) - Talks given about Go and Gonum delivered by Gonum developers.
 * [gonum.github.io](https://github.com/gonum/gonum.github.io) - Gonum website
 
-## How to Contribute
+## Contributing
 
 ### Reporting Bugs
 
-When you encounter a bug, please open an issue on the corresponding repo.  Be as specific as possible.  If you are able to write a test that reproduces the bug, please open a pull instead of an issue.  As a rule we keep all tests OK.
+When you encounter a bug, please open an issue on the corresponding repo.  Start the issue title with the repo/subrepo name, like `stat/distmv: Issue Name`.  Be specific about the environment you encountered the bug in.  If you are able to write a test that reproduces the bug, please include it in the issue.  As a rule we keep all tests OK.
```

Comment:
Probably not. Remember that there are intentions to merge the repos into one.

Also, a PR template - github.com/golang/go has [this](https://github.com/golang/go/tree/master/.github) holding both of them.